prow/cmd/deck/static/script.js

```diff
@@ -492,17 +485,13 @@ function createTimeCell(id, time) {
     var momentTime = moment.unix(time);
     var tid = "time-cell-" + id;
     var main = document.createElement("div");
-    var age = momentTime.fromNow();
-    var timeString = momentTime.format('MMM DD YYYY, HH:MM:SS [GMT]ZZ');
-    main.textContent = timeString;
+    main.textContent = momentTime.format('HH:mm:ss');
```

Comment:
Make this show days if it's older?

    main.TextContent = momentTime.format(momentTime.isBefore(moment().startOf('day')) ? 'MMM DD HH:mm' : 'HH:mm:ss')

Reply:
to be fair this is basically an edge case doesn't in deck since we constantly purge jobs with sinker

Reply:
http://prow.k8s.io/?type=batch has 3 days of jobs

Reply:
done